app/models/filters/filter_base.rb

```diff
@@ -1068,6 +1072,18 @@ def chosen(key)
         cohort_column_housed_date
       when :cohort_column_matched_date
         cohort_column_matched_date
+      when :chronic_status
+        if chronic_status
+          'Yes'
+        elsif chronic_status == false
+          'No'
+        end
```

Comment:
The asymmetry annoys me -- is `chronic_status` a boolean? In which case, I think
```suggestion
        case chronic_status
        when true
          'Yes'
        when 'false
          'No'
        else # This is optional, but you could include it if you prefer
          nil
        end
```
is clearer... If it can be something else, I would prefer something like `if chronic_status.present?`

Or maybe an optional parameter for `ApplicationHelper#yes_no` to determine if it should return a value for `nil`?